src/encryption.ts

```diff
@@ -165,18 +165,15 @@ export function decrypt({
 
   switch (encryptedData.version) {
     case 'x25519-xsalsa20-poly1305': {
-      // string to buffer to UInt8Array
-      const receiverPrivateKeyUint8Array = naclDecodeHex(privateKey);
+      const receiverPrivateKeyUint8Array = Buffer.from(privateKey, 'hex');
```

Comment:
Can we use `@metamask/utils` instead of Node.js globals?

Reply:
Can we address that in a separate PR? This function is already using the native `Buffer` ([here](https://github.com/MetaMask/eth-sig-util/pull/358/files#diff-e15805cccaeceb899db45f9ad7fa014fe8373c838e340e85d29f967afb4d0eb8L258)), so we're not introducing it here.